lib/behaves.rb

```diff
@@ -6,8 +6,13 @@ def implements(*methods)
     @behaviors ||= Set.new(methods)
   end
 
+  def inject_behaviours (&block)
```

Comment:
I think `inject` is succinct enough with less words to type, what do you think?

```ruby
class Hunter
  inject do
    def hunt; end
  end
end
```

Reply:
Also reduces the confusion between US and UK spelling of `behaviors` vs `behaviours`

Reply:
I feel `inject_behaviours` is more clear. When we use `inject` it feel incomplete, `inject_behaviors` is more directly in a sense that is conveys what we are trying to achieve exactly.

Reply:
My concern would be that it could confuse non-native speakers, for example in the comment you had just written, you had spelt it as `behaviours`. That's just an additional overhead.

However, I understand and agree your point about it being clearer too - perhaps we could alias them, what do you think?

Reply:
Not sure @edisonywh lets get some feedback from the others if possible.